server/re/include/irods/msi_genquery2.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef IRODS_MSI_GENQUERY2_HPP
+#define IRODS_MSI_GENQUERY2_HPP
+
+/// \file
+
+struct MsParam;
+struct RuleExecInfo;
+
+/// TODO
+///
+/// \since 4.3.2
+auto msi_genquery2_execute(MsParam* _handle, MsParam* _query_string, RuleExecInfo* _rei) -> int;
+
+/// TODO
+///
+/// \since 4.3.2
+auto msi_genquery2_next_row(MsParam* _handle, RuleExecInfo* _rei) -> int;
+
+/// TODO
+///
+/// \since 4.3.2
+auto msi_genquery2_column(MsParam* _handle, MsParam* _column_index, MsParam* _column_value, RuleExecInfo* _rei) -> int;
+
+/// TODO
+///
+/// \since 4.3.2
+auto msi_genquery2_free(MsParam* _handle, RuleExecInfo* _rei) -> int;
```

Comment:
I think you may be looking at an old commit?

Reply:
I reviewed the changes by clicking "Files changed" in GitHub and I was under the impression that it would be based on the latest commit. Line 24 says "TODO" and it doesn't say "Outdated" so I'm assuming it's the latest. Might be wrong, just commenting on what I see :)

Reply:
I see what happened. I added the documentation to the cpp file instead of the header file. Good catch.

Will fix.

Reply:
Done